spec/std/math_spec.cr

```diff
@@ -1,4 +1,5 @@
 require "spec"
+require "big"
```

Comment:
You can remove this line now.

Reply:
@RX14 I get `Error in spec/std/math_spec.cr:46: undefined constant BigFloat` then